src/modules/job-post/job-post.controller.ts

```diff
@@ -52,4 +52,16 @@ export class JobPostController {
   remove(@Param('id') id: string) {
     return this.jobPostService.remove(+id);
   }
+
+  @UseGuards(JwtAuthGuard)
+  @Get('home/:id/:isDraft')
+  getClientHomePostsAndDrafts(
+    @Param('id') id: number,
+    @Param('isDraft') isDraft?: string,
+  ) {
+    return this.jobPostService.getClientHomePostAndDrafts(
+      id,
+      isDraft === 'true' ? true : false,
```

Comment:
Why can't the isDraft url param be boolean?

Reply:
I could not figure out a way to do it like a boolean param inside query params, since they are only string type